### PR TITLE
fix: Release patch - payment issue

### PR DESCRIPTION
**Pull request by Mukamik**

### CODE REVIEW

**Comment by Shabirmean** [comment]
:zap:  Release Status :zap: </br>
- **Version updated**: Version in the  has been updated to the next **patch** release version.
- **Publish Artifacts:** Check the most recent commit (5d0cf1c1cff87cde32b4f06ad67386bcfc8869bd) for correctness and run the ["pos-publish-release-artifacts-jss"](https://console.cloud.google.com/cloud-build/triggers;region=global?project=point-of-sale-ci) trigger against this branch **[release-patch]** to upload the artifacts _(Jars and Images)_ to Artifact registry.

**Comment by Shabirmean** [comment]
:zap:  Release Status :zap: </br>
- **Version updated**: Version in the  has been updated to the next **patch** release version.
- **Publish Artifacts:** Check the most recent commit (8536467a623badcbbc22f32f90f06f318dfd87a1) for correctness and run the ["pos-publish-release-artifacts-jss"](https://console.cloud.google.com/cloud-build/triggers;region=global?project=point-of-sale-ci) trigger against this branch **[release-patch]** to upload the artifacts _(Jars and Images)_ to Artifact registry.

**Comment by Shabirmean** [comment]
:zap:  Release Status :zap: </br>
- **Version updated**: Version in the  has been updated to the next **patch** release version.
- **Publish Artifacts:** Check the most recent commit (067b39c9af24170481a4bc09d5191195ff60f540) for correctness and run the ["pos-publish-release-artifacts-jss"](https://console.cloud.google.com/cloud-build/triggers;region=global?project=point-of-sale-ci) trigger against this branch **[release-patch]** to upload the artifacts _(Jars and Images)_ to Artifact registry.

File: src/ui/src/components/MainPage.vue [hunk 1]
```diff
@@ -19,7 +19,7 @@
     <div class="row" id="pos">
       <div class="col-md-8 col-md-offset-2">
         <div class="header-container">
-          <h1 class="h1-flex">Point of Sale</h1>
+          <h1 class="h1-flex">Point of Sale!</h1>
```
Comment (bourgeoisor) [inline src/ui/src/components/MainPage.vue:22]: Is that an intentional change?

Comment (Mukamik) [reply src/ui/src/components/MainPage.vue:22]: Yes it is

**Comment by Shabirmean** [comment]
:zap:  Release Status :zap: </br>
- **Version updated**: Version in the  has been updated to the next **patch** release version.
- **Publish Artifacts:** Check the most recent commit (5de64df9f8da2badb91402c03fda05b22f86fbcf) for correctness and run the ["pos-publish-release-artifacts-jss"](https://console.cloud.google.com/cloud-build/triggers;region=global?project=point-of-sale-ci) trigger against this branch **[release-patch]** to upload the artifacts _(Jars and Images)_ to Artifact registry.

**Comment by Shabirmean** [comment]
:zap:  Release Status :zap: </br>
- ~~**Artifacts published**: [Maven JARs and Container images](https://console.cloud.google.com/artifacts/browse/point-of-sale-ci?project=point-of-sale-ci) for release version **v0.3.1-jss** have been published to Artifact Registry~~
- **Manifests updated**: [Release manifests](https://github.com/GoogleCloudPlatform/point-of-sale/tree//k8-manifests/release) have been updated to use the version **v0.3.1-jss** images
- **Version bumped:** Version in the _pom.xml/package.json_ has been bumped () to the snapshot version of the next **patch** release **v**
- **Github release created:** A draft [Github release](https://github.com/GoogleCloudPlatform/point-of-sale/releases) has been created. Inspect and **publish it**
- **Merge PR:** Inspect the latest changes and merge the PR to **jss** to complete the release.

:rocket:

**Review by minherz — APPROVED** [review]
lgtm

**Pull request merged**